### PR TITLE
Importpath

### PR DESCRIPTION
**Pull request by scottnath**
The default syntax for importing .scss files with compass is to remove leading uderscores and the file extension on all .scss files. 
https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/README.md#importpath
I added this section, which is off by default, to allow the imports to be written correctly for compass defaults.

thanks,
Scott

### CODE REVIEW

**Comment by jsahlen** [comment]
Thanks! Do things break if using an explicit filename in the `@import` statements though?

File: index.js [hunk 1]
```diff
@@ -72,17 +86,31 @@ var cssGlobbingPlugin = function(options) {
       files = [];
 
       if (globRegExp.exec(filePattern)) {
-        glob.sync(filePattern, { cwd: path.dirname(filename) }).forEach(function(foundFilename) {
-          if ((options.extensions.indexOf(path.extname(foundFilename)) !== -1)&&(options.ignoreFolders.indexOf(path.dirname(foundFilename))) == -1) {
-            files.push(foundFilename);
+        glob.sync(filePattern, { cwd: path.dirname(filename) }).forEach(function(foundFilePath) {
+          if ((options.extensions.indexOf(path.extname(foundFilePath)) !== -1)&&(options.ignoreFolders.indexOf(path.dirname(foundFilePath))) == -1) {
+
+            var foundFilename = foundFilePath.split('\\').pop().split('/').pop();
```
Comment (jsahlen) [inline index.js]: I think we should defer more to `path` in this block – for example use `path.basename()` on this row, get the extension for replacement with `path.extname()`, join back together with `path.join()` + `path.dirname()`, etc.

File: test/main.js [hunk 2]
```diff
@@ -173,7 +259,7 @@ describe('gulp-css-globbing', function() {
       var options = {}
 
       options.autoReplaceBlock = {
-          onOff:true, 
+          onOff:true,
```
Comment (jsahlen) [inline test/main.js]: Nice ;)

File: index.js [hunk 3]
```diff
@@ -47,6 +54,13 @@ var cssGlobbingPlugin = function(options) {
     if(!options.autoReplaceBlock.globBlockContents) options.autoReplaceBlock.globBlockContents = autoReplaceBlockDefaults.globBlockContents;
   }
 
+  if (!(options.scssImportPath instanceof Object)) {
+    throw new gutil.PluginError(PLUGIN_NAME, 'scss import path needs to be an object');
+  }else{
+    if(typeof options.scssImportPath.leading_underscore === 'undefined') options.scssImportPath.leading_underscore = scssImportPathDefaults.leading_underscore;
+    if(typeof options.scssImportPath.filename_extension === 'undefined') options.scssImportPath.filename_extension = scssImportPathDefaults.filename_extension;
+  }
+
```
Comment (jsahlen) [inline index.js]: I realize that there is no formal styleguide or anything, but I think spaces after `if` statements (e.g. `if (…`) and around curly braces `} else {` make things much easier to read.

**Comment by scottnath** [comment]
@jsahlen made the changes. thanks for the feedback. please let me know if I should adjust anything else.

-scott

**Comment by jsahlen** [comment]
Thanks so much!

**Pull request merged**